### PR TITLE
feat: Image 

### PR DESCRIPTION
**Pull request by shivangisharma01**
image
image
image

### CODE REVIEW

File: components/ImageR/imageR.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.square-image {
+    height: 9rem;
+    width: 9rem;
+    margin: 10px;
+    object-fit: cover;
```
Comment (dev-enforced) [inline components/ImageR/imageR.css:5]: This line is repeated multiple times so you can pass this as a property to your image element in the reset.css file which you might have made for this Library.

File: components/ImageR/imageR.html [hunk 2]
```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="imageR.css" />
+    <link rel="stylesheet" href="/css/styles.css" />
```
Comment (dev-enforced) [inline components/ImageR/imageR.html:8]: Always keep your stylesheets at the bottom of the head to make sure that your styles get applied to the element

File: components/ImageR/imageR.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.round-image {
+    height: 9.5rem;
+    width: 9.5rem;
+    border-radius: 50%;
```
Comment (dev-enforced) [inline components/ImageR/imageR.css:11]: Do pass this value of 50% as a variable in your variables.css file with --border-rd-circle:50% for better reading practises in code

**Review by dev-enforced — APPROVED** [review]
The code looks good to me. Suggested some minor changes which you can add in your code!!
Do deploy this repository on Netlify or Vercel because it generates a live preview of the applications for the pull requests that you create.Good work @shivangisharma01

File: components/ImageR/imageR.html [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+      </div>
+    </div>
+    <footer class="footer">
+      <div class="footer-header">Made by Shivangi Sharma</div>
+      <ul class="footer-list">
+        <li class="list-item-inline">
+          <a
+            class="links"
+            href="https://github.com/Shivangisharma01"
+            target="blank"
```
Comment (sadathshariff) [inline components/ImageR/imageR.html:202]: are you planning to open in the same window since you are using just blank, it will also work

**Review by sadathshariff — APPROVED** [review]
The Code looks Clean and good, Suggested something do check it out

File: components/ImageR/imageR.html (hunk 4 above)
Comment (shivangisharma01) [reply components/ImageR/imageR.html:202]: The target blank will open this link in new tab, that's why I've used it.

File: components/ImageR/imageR.css (hunk 3 above)
Comment (shivangisharma01) [reply components/ImageR/imageR.css:11]: done

File: components/ImageR/imageR.html (hunk 2 above)
Comment (shivangisharma01) [reply components/ImageR/imageR.html:8]: okay

File: components/ImageR/imageR.html (hunk 4 above)
Comment (sadathshariff) [reply components/ImageR/imageR.html:202]: target:_blank opens up in new window and blank opens up in the same window,do check it once

File: components/ImageR/imageR.css (hunk 1 above)
Comment (shivangisharma01) [reply components/ImageR/imageR.css:5]: okay, i'll do that

**Pull request merged**